spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Normalize.scala

```diff
@@ -69,13 +69,49 @@ class Normalize[T: ClassTag](val p: Double, val eps: Double = 1e-10
       } else {
         buffer.resizeAs(inputBuffer).pow(inputBuffer, ev.fromType(p))
       }
-      normp.sum(buffer, 2).add(ev.fromType(eps))
+      // normp.sum(buffer, 2).add(ev.fromType(eps))
```

Comment:
remove it ?

Reply:
The same as above. This is simple and easy to understand. The while loop below is the optimization version.